### ISSUE TITLE
Add support for Files on iOS 11+

### ISSUE THREAD

**Issue by kylemsguy** [yoyofr/modizer#40]
Currently, there's no way to copy files from other apps/Safari to Modizer (the app isn't a share target on iOS 14). The simplest way to allow for this seems to be to expose the Documents directory to the Files app. This seems to be a simple change to the Info.plist (in the case of this repo, `modizer/modizer-info.plist`):

Add the following:
```
<key>LSSupportOpeningDocumentsInPlace</key>
<true/>
```

I was planning on submitting a pull request with the change, but I'm currently unable to get the app on master to run without crashing on startup. As is, the code on master doesn't build on a fresh clone.

**Comment by yoyofr** [comment]
thanks, I will check

**Comment by redbug26** [comment]
Yes, please. We need that ;)

**Comment by yoyofr** [comment]
fixed in v2.9.4

**Comment by kylemsguy** [comment]
I noticed that the share target is working again on v2.9.4, but when I share a zip of MIDIs to it, the file starts playing, but doesn't show up in the database. It would be nice to be able to import and save files via the share target. (Currently the tmp folder is inaccessible)

Furthermore, the app doesn't seem to show up in Files. This would be very helpful as it allows us to directly copy files from Safari's downloads into the app. This is what I originally suggested in the original issue, and it seems that the key I suggested is set to false in the info.plist on master.
